test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
what is this commented out code? is it being saved for later or can it just be removed? We haven't written any tests yet right? so is uncommenting this going to break anything?

Reply:
This is auto generated test code that comes out of the box from rails mini test and I guess we can get rid of the whole mini test package and auto generated test files that will be replaced by Rspec :)